types/src/multihash.rs

```diff
@@ -0,0 +1,214 @@
+use multihash_derive::{Hasher, MultihashDigest};
+use nmt_rs::simple_merkle::tree::MerkleHash;
+use nmt_rs::NamespaceMerkleHasher;
+
+use crate::byzantine::MULTIHASH_SHA256_NAMESPACE_FLAGGED_CODE;
```

Comment:
can we move this constant and other multihash related stuff there?

Reply:
right, moved 👍